docs/deployment/deployK8s-AWS.md

```diff
@@ -5,343 +5,158 @@ title: Prepare AWS EKS for Multi-Region Deployments
-
-         ```shell
-         eksctl create cluster -f ca-central-1.yaml --profile <aws-profile>
-         ```
-
-      * Create the second cluster:
-
-         ```shell
-         eksctl create cluster -f us-west-2.yaml --profile <aws-profile>
-         ```
-
-3. Sign on to the AWS console and navigate to the **VPC** service.  Select **Your VPCs** (under Virtual Private Cloud), and record the VPC details for the clusters that were just created.
-
-      > Make note of the `VpcId` values for the `ca-central-1` and `us-west-2` VPCs for use in subsequent steps.
-
-4. Set up VPC peering between the two clusters.
-
-      Create a peering connection from the cluster in the `us-west-2` region to the cluster in the `ca-central-1` region, using the VPC Dashboard as in the previous step.
-
-      * At the top right of the page, select the **Oregon** (us-west-2) region.
-
-      * Select **Peering Connections** and click **Create Peering Connection**.
-
-      * Assign a unique name for the peering connection (for example, us-west-2-to-ca-central-1).
-
-      * Under the **Select a local VPC to peer with** section, enter the `VpcId` value for the `us-west-2` VPC.
-
-      * In the **Select another VPC to peer with** list, select **My account** --> **Another region** --> **Canada Central** (ca-central-1).
-
-      * Under the **VPC (Accepter)** section, enter the `VpcId` value for the `ca-central-1` region.
+1. Create VPCs.
 
-      * Click **Create Peering Connection** and when you receive a confirmation message, click **OK** to continue.
+    * Sign on to the AWS console and navigate to the **VPC** service.
 
-      * At the top right of the page, change the region to **Canada Central**.
+    * Toggle to the `eu-west-1` region.
 
-      * Select **Peering Connections**.
+    * Select **Your VPCs** (under Virtual Private Cloud) and click **Create VPC**
 
-         > The peering connection status for `us-west-2` will show as `Pending Acceptance`.
+    * Add a name tag, such as `demo-vpc-eu-west-1`
 
-      * Select the `ca-central-1` connection, click the **Actions** list, and select **Accept Request** and confirm when prompted.
+    * Add a IPv4 CIDR, such as `10.0.0.0/16`
 
-         > The VPC peering connection status should now show as `Active`.
+    * Click **Create VPC**.
 
-5. Get the subnet information for each cluster node.
+    > Make note of the `VpcId` and `IPv4 CIDR` values for the `eu-west-1` and `us-east-1` VPCs for use in subsequent steps.
 
-      Each cluster node uses a different subnet, so there are three subnets assigned to each VPC. The information displayed contains the subnet ID for each subnet. Use the subnet IDs in the subsequent step to get the associated routing tables.
+    * Repeat this step in `us-east-1` region.
 
-      * At the top right of the page, change the region to **Oregon**.
+2.  Create the transit gateway for each region your deployment is being hosted on. Toggle to the `eu-west-1` region.
```

Comment:
.... for each region _**on which your deployment is being hosted**_.  Toggle ....